TACC-Cloud/geoapi
# task/WG-258: update project metadata on map creation or deletion

## Overview: ##

Update project metadata on map creation or deletion

## Related Jira tickets: ##

* [WG-258](https://tacc-main.atlassian.net/browse/WG-258)

## Testing Steps: ##
1. Run with hazmapper 
2. Create a map connected to a designsafe project
3. Check that the project metadata has hazmapperMaps that include this new map


## Note ##

waiting on https://github.com/DesignSafe-CI/portal/pull/1255 to be merged and deployed to designsafenext
